EventFilter/L1TRawToDigi/plugins/implementations_stage2/GMTCollections.cc

```diff
@@ -10,11 +10,28 @@ namespace l1t {
          event_.put(std::move(regionalMuonCandsOMTF_), "OMTF");
          event_.put(std::move(regionalMuonCandsEMTF_), "EMTF");
          event_.put(std::move(muons_), "Muon");
+         event_.put(std::move(muonsSet2_), "MuonSet2");
```

Comment:
why not just a muons_[6]? then loops could be conveniently be used in a number of places below?